### ISSUE TITLE
current 1.00.0 CRAN release does not match JOSS 1.00.0 release

### ISSUE THREAD

**Issue by cjyetman** [vincentarelbundock/countrycode#194]
Something is very strange.

current CRAN release is versioned as 1.00.0 and published 2018-02-11
https://CRAN.R-project.org/package=countrycode
it does **not** include "vdem", "gwc", and "gwn" destination codes

there's a 0.08 tagged release on Github from 2018-08-08
https://github.com/vincentarelbundock/countrycode/releases/tag/0.08
it **does** include "vdem", "gwc", and "gwn" destination codes

and then there's the JOSS tagged release (1.00.0) on Github from 2018-08-08
https://github.com/vincentarelbundock/countrycode/releases/tag/1.00.0-joss
which also **does** include "vdem", "gwc", and "gwn" destination codes

obviously I'm a bit confused at the moment, so sorry if I'm missing something obvious

I guess the 0.08 tagged version was meant to be submitted to CRAN as 1.01.0? And the JOSS release should have been versioned as 1.01.0? And whatever 1.01.0 is has yet to be submitted to CRAN ever?

**Comment by vincentarelbundock** [comment]
You know, I think this is a *mea culpa* kind of situation. My understanding was that JOSS needed a version number for a package that was both (a) released, and (b) available on github. The ideal solution would have been to make a new CRAN release just before submission to JOSS. However, I either didn't realize there had been changes to master, or got too lazy.

So here we are. How about we just make a new tag to clearly indicate that master is a new version?

**Comment by cjyetman** [comment]
There have been some rather significant changes since the 2018-02-11 CRAN release (like the GW codes), so if it's not too much trouble (hahaha), a new CRAN submission with a bumped version number would probably be ideal.

But at a minimum, we should probably bump the version number in master here. If you follow [Hadley's advice](http://r-pkgs.had.co.nz/release.html), it should probably be 1.00.0.9000

**Comment by vincentarelbundock** [comment]
I like the idea of 9000 for dev version, but it shouldn't be too hard to put together a new CRAN version. I'll give it a shot tonight.

**Comment by vincentarelbundock** [comment]
Is this serious enough of a bug to delay release until we have a fix?

https://github.com/vincentarelbundock/countrycode/issues/192

**Comment by cjyetman** [comment]
#192 is fixed. Another related issue is now opened #196, but I suspect that is a **much** less likely scenario, so I wouldn't hold the release on it.

**Comment by vincentarelbundock** [comment]
Submitted to CRAN. Waiting for the inevitable request for changes.

Here's the tag: https://github.com/vincentarelbundock/countrycode/releases/tag/1.01.0

Closing this now as I'll respond to CRAN requests as they come. 
Thanks for the help!

**Comment by vincentarelbundock** [comment]
FYI, CRAN bounced my submission because it no longer allows leading zeroes in version numbers.

I have update the version number to 1.1.0. I checked, and we don't mention version 1.00.0 anywhere in the paper, so I updated the release tag to 1.0.0 as well on github. The only remaining incongruity is that if people check the CRAN version history, they'll see 1.00.0, but that seems like a really minor issue.

We're now set for life in terms of version numbers. And I like the dev version idea, so I pushed a commit with a change to DESCRIPTION now.
